libopae/pluginmgr.c

```diff
@@ -62,6 +62,9 @@ static platform_data platform_data_table[] = {
 	{ 0x8086, 0x09c4, "libxfpga.so", 0 },
 	{ 0x8086, 0x09c5, "libxfpga.so", 0 },
 	{ 0x8086, 0x0b2b, "libxfpga.so", 0 },
+	{ 0x8086, 0x0b2c, "libxfpga.so", 0 },
```

Comment:
These new vendor ids are for darby/vista creek etc.?

Reply:
This is a merge from master.